### PR TITLE
Feature/add responds to ping test

### PR DESCRIPTION
**Pull request by Pixep**

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -14,5 +14,5 @@ jobs:
       run: make docker-build
     - name: Run 'check' in Docker with sample files
       run: docker run --rm witekio/pluma-automation:latest pluma check -c /etc/pluma/pluma.yml -t /etc/pluma/pluma-target.yml
-    - name: Run tests from Docker
-      run: docker run --rm -e PLUMA_ENV=CI witekio/pluma-automation:latest bash -c "cd /root/pluma && make test"
+    # - name: Run tests from Docker
```
Comment (mjftw) [inline .github/workflows/docker-image.yml:17]: Do we definitely need to disable running the tests in Docker?

**Review by mjftw — CHANGES_REQUESTED** [review]
Code all looks good, just unsure about disabling the docker tests.

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (Pixep) [reply .github/workflows/docker-image.yml]: These are unstable on the CI, and already on master once rebase :). This is something we will have to investigate

Comment (Pixep) [reply .github/workflows/docker-image.yml]: https://github.com/Witekio/pluma-automation/issues/69

**Comment by Pixep** [comment]
Branch content approved, merging ahead of tomorrow's presentation

**Pull request merged**